### ISSUE TITLE
Use of Redis causes "job priority is not supported in the driver" error

### ISSUE THREAD

**Issue by john-henry** [putyourlightson/craft-blitz#400]
I'm getting this now when I upload an Asset or save any changed Entry/Element. I have narrowed it down to Blitz

I only installed Redis this morning and all of that works fine. I only use it for queues

```
 'redis' => [
        'class' => yii\redis\Connection::class,
        'hostname' => 'localhost',
        'port' => 6379,
        'database' => 1,
      ],
    'queue' => [
        'class' => yii\queue\redis\Queue::class,
        'redis' => 'redis', 
        'channel' => 'my_queue', 
    ],
```
`"yiisoft/yii2-redis": "^2.0"`

Here is Sentry error https://sentry.io/share/issue/e483de1948d04ac4bdf28030f73f9990/

I do use `enableTemplateCaching` set to true but I have tested on or off and same issue uploading

I also tried setting priority to null in Blitz config but that didn't work either

Let me know if you need any more details

### Versions
- Plugin version:3.12.2
- Craft version:3.7.39

**Comment by bencroker** [comment]
Thanks for reporting this bug. It was caused by a regression introduced in https://github.com/putyourlightson/craft-blitz/commit/694d170b03e8c1cb254ddf3258d4b948259b98c6 and released in version 3.12.0. 
Fixed in https://github.com/putyourlightson/craft-blitz/commit/300980168d9861437f1251596be225e8a47faf91 for the next release.

**Comment by bencroker** [comment]
Released in version 3.12.3.
